vuepress/docs/next/tutorials/create/ms/add-api-claim.md

```diff
@@ -62,7 +62,7 @@ ent bundle run simple-node-ms
 * Access <http://localhost:8081/api/hello> to see a `Hello World!` response with a timestamp. 
 * Access <http://localhost:8081/api/health> to see a standard `{"status:"UP"}` response.
 
-In local development, the run command can be used to modify the port to run multiple microservices, but in production, microservices must run on port 8081. 
+In local development, custom commands can be used to modify ports to run each microservice, but in production, microservices must run on port 8081. 
```

Comment:
i think this is less clarified... prefer nathan's tweak to your original wording: In local development, the run command can be used to modify the port for a microservice, but in production microservices must run on port 8081.

Reply:
I wanted to include cases where there are multiple microservices because that's when this applies, not when there is just one

Reply:
In local development, the run command can be used to modify the port number so that multiple microservices can run in parallel on separate ports, but in production microservices must run on port 8081